docs/using-mycroft-ai/customizations/wake-word.md

```diff
@@ -181,6 +187,10 @@ Mycroft will respond
 
 To return to Precise, speak:
 
+> Hey Mycroft, set the Listener to default
```

Comment:
since 2018 PocketSphinx is no longer the default, so switch this appropriately.

Reply:
Interestingly this currently seems to set the listener to Pocketsphinx so we should update that in the Configuration Skill

Reply:
(let me know how you'd like to handle; a sort of chicken/egg thing here or both should be fixed at the same time)

Reply:
No all good, I'm fixing up the Skill now. Just adding tests to that Skill at the same time as it's a bit light on.